src/core/core.controller.js

```diff
@@ -249,6 +249,10 @@ class Chart {
     // After init plugin notification
     this.notifyPlugins('afterInit');
 
+    if (this.options.fill && this._plugins._init.filter(p => p.plugin.id === 'filler').length === 0) {
```

Comment:
How about keeping the `PluginService` internals internal?
Also maybe add a public `isPluginEnabled` (or something) to `core.controller`, so there is an interface for extensions to do the same.

Something like:
```js
isEnabled(pluginId) {
  this._init.filter(p => p.plugin.id === 'filler').length !== 0
}
```